meteor/client/ui/Prompter/PrompterView.tsx

```diff
@@ -113,14 +117,21 @@ export class PrompterViewInner extends MeteorReactComponent<Translated<IProps &
 			})
 		}
 
-		let playlistId: RundownPlaylistId =
-			(this.props.rundownPlaylist && this.props.rundownPlaylist._id) || protectString('')
-
 		this.autorun(() => {
-			let playlist = RundownPlaylists.findOne(playlistId)
-			if (playlistId) {
+			let playlist = RundownPlaylists.findOne(
+				{
+					studioId: this.props.studioId,
+					active: true,
+				},
+				{
+					fields: {
+						_id: 1,
+					},
+				}
+			)
+			if (playlist?._id) {
 				this.subscribe(PubSub.rundowns, {
-					playlistId: playlistId,
+					playlistId: playlist?._id,
```

Comment:
minor note: the optional chaining `?.` is unnecessary here.

Reply:
Removed the optional chaining.